annotations/exports.py

```diff
@@ -1,4 +1,5 @@
 from django.db.models import Count, Max
+import numbers
```

Comment:
Why do we need this import? :-)

Reply:
oops, this was a teaser for the csv export ;)